test/unittests/iotools/converters/adni_to_bids/adni_modalities/test_adni_fmap.py

```diff
@@ -0,0 +1,306 @@
+import json
+
+import pytest
+
+from clinica.iotools.converters.adni_to_bids.adni_modalities.adni_fmap import (
+    BIDSFMAPCase,
+)
+
+
+@pytest.mark.parametrize(
+    "case, input_value, expected",
+    [
+        (BIDSFMAPCase.ONE_PHASE_TWO_MAGNITUDES, "e2", "magnitude2"),
+        (BIDSFMAPCase.ONE_PHASE_TWO_MAGNITUDES, "e2_ph", "phasediff"),
+        (BIDSFMAPCase.TWO_PHASES_TWO_MAGNITUDES, "e1", "magnitude1"),
+        (BIDSFMAPCase.TWO_PHASES_TWO_MAGNITUDES, "e2_ph", "phase2"),
+    ],
+)
+def test_phase_magnitude_renamer_success(case, input_value, expected):
+    from clinica.iotools.converters.adni_to_bids.adni_modalities.adni_fmap import (
+        phase_magnitude_renamer,
+    )
+
+    assert phase_magnitude_renamer(input_value, case) == expected
+
+
+@pytest.mark.parametrize(
+    "case, input_value",
+    [
+        (BIDSFMAPCase.ONE_PHASE_TWO_MAGNITUDES, "_e2"),
+        (BIDSFMAPCase.ONE_PHASE_TWO_MAGNITUDES, "e7"),
+        (BIDSFMAPCase.ONE_PHASE_TWO_MAGNITUDES, "ee1"),
+        (BIDSFMAPCase.TWO_PHASES_TWO_MAGNITUDES, "e2ph"),
+        (BIDSFMAPCase.TWO_PHASES_TWO_MAGNITUDES, " e2"),
+        (BIDSFMAPCase.TWO_PHASES_TWO_MAGNITUDES, "ea"),
+        (BIDSFMAPCase.TWO_PHASES_TWO_MAGNITUDES, "21"),
+    ],
+)
+def test_phase_magnitude_renamer_value_error(case, input_value):
+    from clinica.iotools.converters.adni_to_bids.adni_modalities.adni_fmap import (
+        phase_magnitude_renamer,
+    )
+
+    with pytest.raises(
+        ValueError, match=f"Extension {input_value} not taken in charge."
+    ):
+        phase_magnitude_renamer(input_value, case)
+
+
+@pytest.mark.parametrize(
+    "case, input_value",
+    [
+        (BIDSFMAPCase.DIRECT_FIELDMAPS, "e2"),
+        (BIDSFMAPCase.NOT_SUPPORTED, "e1"),
+        (BIDSFMAPCase.EMPTY_FOLDER, "toto"),
```

Comment:
`toto` is very French. ;) 
Usually the convention is to name these "foo", "bar", and "baz".

```suggestion
        (BIDSFMAPCase.EMPTY_FOLDER, "foo"),
```